tests/expr/cat/get_categories_test.py

```diff
@@ -11,11 +12,16 @@
 
 
 def test_get_categories(constructor: Any) -> None:
+    if "pyarrow_table" in str(constructor):
+        pytest.xfail()
```

Comment:
I think we need to use `request.applymarkers(pytest.mark.xfail)`, not `pytest.xfail`, otherwise the rest of the test would be skipped https://github.com/pytest-dev/pytest/issues/7071

Reply:
Oh wow! I just learned something new! Thanks a lot!